AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -120,34 +122,35 @@
             <div class="col-sm-12">
                 @if (!User.IsSignedIn())
                 {
-                    @if (!Model.IsClosed)
-                    {
-                        <h2>
-                            <a asp-action="Login" asp-controller="Account" asp-route-ReturnUrl="/Activity/@Model.Id" class="btn btn-volunteer">Volunteer</a>
-                        </h2>
-                    }
-                    else {
-                        <h2>This activity is closed</h2>
-                    }
+                    if (!Model.IsClosed)
+                     {
+                         <h2>
+                             <a asp-action="Login" asp-controller="Account" asp-route-ReturnUrl="/Activity/@Model.Id" class="btn btn-primary  btn-volunteer">Volunteer</a>
+                         </h2>
+                     }
+                     else
+                     {
+                         <h2>This activity is closed</h2>
+                     }
                 }
                 else
                 {
-                    @if (!Model.IsClosed)
-                    {
-                        <h2>
-                            <a href="#" data-toggle="modal" data-target="#volunteerForm" class="btn btn-volunteer" data-bind="visible: !enrolled()" style="display: none">Volunteer</a>
-                        </h2>
-                    }
-                    else
-                    {
-                        <h2 data-bind="visible: !enrolled()"> This activity is closed</h2>
-                    }
+                    if (!Model.IsClosed)
+                     {
+                         <h2>
+                             <button class="btn btn-primary btn-volunteer" data-bind="visible: !enrolled(), click: signupForActivity" style="display: none">Volunteer<br /> for this Activity</button>

```

Comment:
@mheggeseth I'll change the color as suggested. I agree that the button should be the same whether signed in or not.  Regarding the wording, I made the change in wording specifically in response to @tonysurma's request in #502:

**`Volunteer link is present and worded with clarity not just 'volunteer' but something like 'Volunteer for this Campaign'`**

I'll be glad to change it to whatever is decided.  I'll let you make the call.